src/docs/content/utils/DynamicCode.jsx

```diff
@@ -0,0 +1,18 @@
+"use client";
```

Comment:
```tsx
import * as React from "react";

export function DynamicCode({ textContent, installName, children }: {textContent: string, installName: string, children: React.ReactNode}) {
  const ref = React.useRef<HTMLDivElement | null>(null);

  React.useEffect(() => {
    if (ref.current) {
      const elements = ref.current.querySelectorAll("span.line span");
      elements.forEach((el)=> {if(el.textContent === textContent){
        el.textContent = ` [options] ${installName}`;
      }})
    }
  }, [installName, textContent]);
  
  return <div ref={ref}>{children}</div>;
}
```

Reply:
This is very very subject to how Shiki parses that code

Reply:
yes, it indeed is, but if we only use it here I feel like it is worth it than just having the same text on 8 places.

The code is updated now